core/block.js

```diff
@@ -40,7 +40,6 @@ goog.require('Blockly.utils.Coordinate');
 goog.require('Blockly.utils.object');
 goog.require('Blockly.fieldRegistry');
 goog.require('Blockly.utils.string');
-goog.require('Blockly.Warning');
```

Comment:
Maybe leave this one here for now. If you remove it you'll have to take care of checking if it exists when we need it.